src/pages/settings/Profile/PersonalDetails/AddressPage.js

```diff
@@ -52,179 +52,159 @@ const defaultProps = {
     },
 };
 
-class AddressPage extends Component {
-    constructor(props) {
-        super(props);
-
-        this.validate = this.validate.bind(this);
-        this.updateAddress = this.updateAddress.bind(this);
-        this.onCountryUpdate = this.onCountryUpdate.bind(this);
-
-        const currentCountry = lodashGet(this.props.privatePersonalDetails, 'address.country') || '';
-        const currentCountryISO = PersonalDetails.getCountryISO(currentCountry) || CONST.COUNTRY.US;
-        const zipSampleFormat = lodashGet(CONST.COUNTRY_ZIP_REGEX_DATA, [currentCountryISO, 'samples'], '');
-        this.state = {
-            isUsaForm: currentCountry === CONST.COUNTRY.US || currentCountry === CONST.USA_COUNTRY_NAME,
-            zipFormat: this.props.translate('common.zipCodeExampleFormat', {zipSampleFormat}),
-        };
-    }
+/**
+ * @param {Function} translate - translate function
+ * @param {Boolean} isUsaForm - selected country ISO code is US
```

Comment:
NAB since you're just moving this

```suggestion
 * @param {Boolean} isUSAForm - selected country ISO code is US
```